README.Rmd

```diff
@@ -129,6 +129,20 @@ Compared to the corresponding base functions, readr functions:
   automatically guessing the delimiter, whether or not the file has a
   header, and how many lines to skip.
 
+
+## Versions
```

Comment:
In this section, can you lean more heavily on the vocabulary we used for this subject in the column type guessing vignette?

Specifically, the notion of **edition** has to be made super clear and currently isn't mentioned here.

I don't think we need to actually link to that vignette, from here, once we get the wording right / complete.

Reply:
I've added more content from the column type guessing vignette that I thought was relevant. Let me know what you think!